.gitignore

```diff
@@ -7,4 +7,3 @@ cypress/screenshots
 cypress/fixtures
 cypress/plugins
 cypress/support
-dist
```

Comment:
I would advise against having `/dist` in public repo in this case. We would need to ensure that it is always up to date with whatever is in master, which would require a precommit hook or something.

Reply:
you are right, thank you